src/Tizen.WebView/Tizen.WebView/Settings.cs

```diff
@@ -98,5 +115,75 @@ public int DefaultFontSize
                 Interop.ChromiumEwk.ewk_settings_default_font_size_set(_handle, value);
             }
         }
+ 
+        /// <summary>
+        /// Whether the script can open windows.
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
+        public bool ScriptsCanOpenWindows
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_settings_scripts_can_open_windows_get(_handle);
+            }
+
+            set
+            {
+                Interop.ChromiumEwk.ewk_settings_scripts_can_open_windows_set(_handle, value);
+            }
+        }
+
+        /// <summary>
+        /// Whether the view port meta tag enabled.
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
+        public bool ViewPortMetaTagEnabled
```

Comment:
What is `ViewPortMetaTag`? 
Would you explain more details?

I want to more summary

Reply:
view port meta tag is used to control the layout on the device.
https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag

Reply:
Done